space-cli/cmd/modules/addons/database.go

```diff
@@ -106,10 +145,69 @@ func addDatabase(dbtype, username, password, alias, version string) error {
 		return utils.LogError("Could not save hosts file after updating add on containers", err)
 	}
 
-	utils.LogInfo(fmt.Sprintf("Started database (%s) with alias (%s)", dbtype, alias))
```

Comment:
Let this log be. We have started the database after all. Add another info log just below it saying you are applying it to the database config.